React-Homeworks/src/components/Footer/SocialMedia/SocialMedia.jsx

```diff
@@ -0,0 +1,11 @@
+
+import "./SocialMedia.css";
+function SocialMedia() {
```

Comment:
Quite a weird name for the component. This name says nothing for the first glance